### PR TITLE
decode form source

### PR DESCRIPTION
**Pull request by nickpell**
https://sentry.io/dimagi/commcarehq/issues/785346194
https://sentry.io/dimagi/commcarehq/issues/785431599/

### CODE REVIEW

**Comment by Rohit25negi** [comment]
Seen some 500s which can be resolved by this

**Review by Rohit25negi — APPROVED** [review]

**Review by kaapstorm — APPROVED** [review]

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -718,6 +718,8 @@ def __get__(self, form, form_cls):
             source = ''
         else:
             source = app.lazy_fetch_attachment(filename)
+            if isinstance(source, bytes):
```
Comment (millerdev) [inline corehq/apps/app_manager/models.py]: This should always be bytes now, right? So the conditional is unnecessary.

Comment (nickpell) [reply corehq/apps/app_manager/models.py]: That's what I thought ... but in tests ```__set__``` is used with unicode, which then causes ```lazy_fetch_attachment``` to return unicode.  I plan on cleaning up these tests, but these simple changes should resolve the P3 quickly.

Comment (nickpell) [reply corehq/apps/app_manager/models.py]: More specifically, I think it's reasonable to have ```source``` always get and set unicode, but we need to encode before putting the attachment.  That seems like the right boundaries for the unicode sandwich.

**Pull request merged**